packages/components/input/src/input.vue

```diff
@@ -56,6 +56,7 @@
           :aria-label="label"
           :placeholder="placeholder"
           :style="inputStyle"
+          :form="props.form"
```

Comment:
## Suggest

I found a part that behaves differently from the previous one.

### Before

<img width="1246" alt="image" src="https://user-images.githubusercontent.com/27342882/195809625-4aaa6a35-bcf7-4249-b9a6-97902eaba6bb.png">

### after

<img width="1247" alt="image" src="https://user-images.githubusercontent.com/27342882/195810204-7e565fca-6e1d-4b5d-9f7b-8430ab26251f.png">

The job is not visible in the url.

## Resolve

The easiest way to fix this is to change the props default to `undefined`.

<img width="1235" alt="image" src="https://user-images.githubusercontent.com/27342882/195810740-ba97474f-93ea-4501-b11a-deb34b88f53b.png">

Reply:
Thanks. I think change the prop's default  to `undefined` is a reasonable way.